ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/vlan-interfaces.j2

```diff
@@ -1,155 +1,169 @@
+{%         if vlan_interfaces[vlan_interface].ipv6_nd_managed_config_flag is arista.avd.defined(true) %}
    ipv6 nd managed-config-flag
 {%         endif %}
-{%         if vlan_interfaces[vlan_interface].ipv6_nd_prefixes is defined and vlan_interfaces[vlan_interface].ipv6_nd_prefixes is not none %}
+{%         if vlan_interfaces[vlan_interface].ipv6_nd_prefixes is arista.avd.defined %}
 {%             for prefix in vlan_interfaces[vlan_interface].ipv6_nd_prefixes %}
-   ipv6 nd prefix {{ prefix }} {% if vlan_interfaces[vlan_interface].ipv6_nd_prefixes[prefix].valid_lifetime is defined and vlan_interfaces[vlan_interface].ipv6_nd_prefixes[prefix].valid_lifetime is not none %}{{ vlan_interfaces[vlan_interface].ipv6_nd_prefixes[prefix].valid_lifetime }} {% endif %}{% if vlan_interfaces[vlan_interface].ipv6_nd_prefixes[prefix].preferred_lifetime is defined and vlan_interfaces[vlan_interface].ipv6_nd_prefixes[prefix].preferred_lifetime is not none %}{{ vlan_interfaces[vlan_interface].ipv6_nd_prefixes[prefix].preferred_lifetime }} {% endif %}{% if vlan_interfaces[vlan_interface].ipv6_nd_prefixes[prefix].no_autoconfig_flag is defined and vlan_interfaces[vlan_interface].ipv6_nd_prefixes[prefix].no_autoconfig_flag == true %}no-autoconfig{% endif %}
-
+{%                 set ipv6_nd_prefix_cli = "ipv6 nd prefix " ~ prefix %}
+{%                 if vlan_interfaces[vlan_interface].ipv6_nd_prefixes[prefix].valid_lifetime is arista.avd.defined %}
+{%                     set ipv6_nd_prefix_cli = ipv6_nd_prefix_cli ~ " " ~ vlan_interfaces[vlan_interface].ipv6_nd_prefixes[prefix].valid_lifetime %}
+{%                 endif %}
+{%                 if vlan_interfaces[vlan_interface].ipv6_nd_prefixes[prefix].preferred_lifetime is arista.avd.defined %}
+{%                     set ipv6_nd_prefix_cli = ipv6_nd_prefix_cli ~ " " ~ vlan_interfaces[vlan_interface].ipv6_nd_prefixes[prefix].preferred_lifetime %}
+{%                 endif %}
+{%                 if vlan_interfaces[vlan_interface].ipv6_nd_prefixes[prefix].no_autoconfig_flag is arista.avd.defined(true) %}
+{%                     set ipv6_nd_prefix_cli = ipv6_nd_prefix_cli ~ " no-autoconfig" %}
+{%                 endif %}
+   {{ ipv6_nd_prefix_cli }}
 {%             endfor %}
 {%         endif %}
-{%         if vlan_interfaces[vlan_interface].multicast.ipv4.source_route_export.enabled is defined and vlan_interfaces[vlan_interface].multicast.ipv4.source_route_export.enabled == true %}
-   multicast ipv4 source route export{% if vlan_interfaces[vlan_interface].multicast.ipv4.source_route_export.administrative_distance is defined and vlan_interfaces[vlan_interface].multicast.ipv4.source_route_export.administrative_distance is not none %} {{ vlan_interfaces[vlan_interface].multicast.ipv4.source_route_export.administrative_distance }}{% endif %}
-
+{%         if vlan_interfaces[vlan_interface].multicast.ipv4.source_route_export.enabled is arista.avd.defined(true) %}
```

Comment:
Missing check for `vlan_interfaces[vlan_interface].multicast` and `vlan_interfaces[vlan_interface].multicast.ipv4` and `vlan_interfaces[vlan_interface].multicast.ipv4.source_route_export`